### PR TITLE
 make the getbook-handler in handler page

### PR DESCRIPTION
**Pull request by hibaakel**

### CODE REVIEW

File: src/handlers.js [hunk 1]
```diff
@@ -1,10 +1,12 @@
 var fs = require('fs');
 var path = require('path');
+const connPool = require('./db_connection.js');
```
Comment (jwld) [inline src/handlers.js]: inconsistent use of const and var is baaad

Comment (MohamedOmarii) [reply src/handlers.js]: :+1:  :+1:

File: src/handlers.js [hunk 2]
```diff
@@ -22,7 +24,18 @@ handlers.addReservation = function(req, res) {
 }
 
 handlers.getBooks = function(req, res) {
-
+    //connect to the db
+    // make query to get all the books
+    connPool.query("SELECT * FROM books", function(err,results) {
```
Comment (jwld) [inline src/handlers.js]: double quotes (should be single)

Comment (JWLD) [inline src/handlers.js]: (should be singles)

Comment (jwld) [reply src/handlers.js]: also add a space between err and results

File: src/handlers.js [hunk 3]
```diff
@@ -22,7 +24,18 @@ handlers.addReservation = function(req, res) {
...
+        if (err) {
+            res.writehead(500, headers.plain);
+            res.end('err with database query');
+        }
+        else {
+            res.writehead(200,headers.json);
```
Comment (jwld) [inline src/handlers.js]: need a space between 200 and headers.json

File: src/handlers.js (hunk 1 above)
Comment (jwld) [reply src/handlers.js]: also, the path to db_connection is incorrect

File: src/handlers.js [hunk 4]
```diff
@@ -1,10 +1,12 @@
 var fs = require('fs');
 var path = require('path');
+const connPool = require('./db_connection.js');
 
 var headers = {
     'html' : {'content-type' : 'text/html'},
     'css' : {'content-type' : 'text/css'},
-    'js' : {'content-type' : 'text/javascript'}
+    'js' : {'content-type' : 'application/javascript'},
+    'json':{'content-type' : 'application/json'}
```
Comment (jwld) [inline src/handlers.js]: add spacing around colon after json on line 9

File: src/handlers.js [hunk 5]
```diff
@@ -22,7 +24,18 @@ handlers.addReservation = function(req, res) {
 }
 
 handlers.getBooks = function(req, res) {
-
+    //connect to the db
+    // make query to get all the books
+    connPool.query("SELECT * FROM books", function(err,results) {
+        if (err) {
+            res.writehead(500, headers.plain);
```
Comment (jwld) [inline src/handlers.js]: headers.plain doesn't exist in the headers object

**Pull request merged**